sn_client/src/files/api.rs

```diff
@@ -41,7 +41,11 @@ pub struct FilesApi {
     wallet_dir: PathBuf,
 }
 
-type ChunkFileResult = Result<(XorName, u64, Vec<(XorName, PathBuf)>)>;
+/// Result of the resulting data address, the DataMap Chunk, filesize and the paths of the chunks to be stored.
+/// If the DataMapChunk exists and is not stored on the network, then it will not be accessible at this address.
+///
+/// This is the (file xorname, datamap_data, filesize, and chunks)
```

Comment:
`file xorname` means the `chunk_name of the DataMap chunk` right ?

Reply:
clarified